main.sh

```diff
@@ -138,28 +136,17 @@ create_pool(){
 #Start daos servers
 start_server(){
     echo -e "\nCMD: Starting server...\n"
-    rm -r server_output.log
-    cmd="orterun  --np $DAOS_SERVERS --hostfile Log/$SLURM_JOB_ID/daos_server_hostlist -x CPATH -x PATH -x LD_LIBRARY_PATH 
-    --enable-recovery daos_server start -i -o $DAOS_SERVER_YAML  
-    --recreate-superblocks 2>&1 | tee server_output.log"
+    cmd="clush --hostfile Log/$SLURM_JOB_ID/daos_server_hostlist \"
+    export PATH=$PATH; export LD_LIBRARY_PATH=$LD_LIBRARY_PATH; export CPATH=$CPATH;
+    export DAOS_DISABLE_REQ_FWD=1;
+    daos_server start -i -o $DAOS_SERVER_YAML  
+    --recreate-superblocks
+    \" 2>&1 "
     echo $cmd
     echo
     eval $cmd &
-    TIMEOUT=0
-    while :
-        do
-        VAL=`cat server_output.log | grep "started on rank" | wc -l`
-        if (($TIMEOUT > 30)); then
-            echo "FAIL: Failed to start DAOS servers"
-            exit 1
-        elif (($DAOS_SERVERS == $VAL));then
-            break
-        else
-            echo "Waiting to start all servers....."
-            sleep 10
-            let "TIMEOUT++"
-        fi
-    done
+
+    sleep 60;
 }
```

Comment:
Why are these checks removed? It was a good way checking the servers actually started. We do need a check to verify server start after launching incase on some nodes servers failed to start.

Reply:
This check is no longer reliable with clush. The stdout may not necessarily be completely flushed to server_output.log.
The more reliable way is to use dmg system query but that will require some work. Probably require a separate ticket for it, as the servers may not join in sequential order. Will add a comment for future work.

Reply:
This block was used to find server startup time in the timing test.  Let's work on the dmg system query and keep timing measurement also a part of this script in future.

Reply:
Yeah 'dmg system query' should be a good one if clush doesn't report complete info back. But I am a bit perplexed proceeding without verification as we had one before. When dealing with large scale clusters it will give way to overlooking such things easily.